e2e/test/scenarios/admin/i18n/content-translation/upload.cy.spec.ts

```diff
@@ -162,6 +163,23 @@ describe("scenarios > admin > localization > content translation", () => {
           );
         });
       });
+
+      it("rejects a CSV upload that is too big", () => {
+        cy.visit("/admin/settings/localization");
+        cy.get("#content-translation-dictionary-upload-input").selectFile(
+          {
+            contents: Cypress.Buffer.from(
+              generateLargeCSV({ sizeInMebibytes: 2.5 }),
+            ),
+            fileName: "file.csv",
+            mimeType: "text/csv",
+          },
+          { force: true },
+        );
+        cy.findByTestId("content-localization-setting").findByText(
+          "Upload failed",
+        );
```

Comment:
I can't check this, but i hope the error message is more verbose and explains the rejection reason.